cli/src/main.rs

```diff
@@ -63,6 +63,11 @@ pub struct Cli {
 	#[clap(long, env, default_value = "false", parse(try_from_str))]
 	pub gmail_use_api: bool,
 
+	/// For Outlook/Office 365 email addresses, use Outlook's API instead of
```

Comment:
I have a question about the name "Outlook". What's the official name of their email subscription service for businesses? I always thought it was "Microsoft 365", but it seems "Office 365" also exists, or is the latter only referring to Word/Excel/PPT..?

I would propose to consolidate around one name, e.g.:
```suggestion
	/// For Microsoft 365 email addresses, use OneDrive's API instead of
```